cropduster/fields.py

```diff
@@ -28,13 +29,62 @@ def sizes(self):
         else:
             return self.field.db_field.sizes
 
-    def regenerate_crops(self):
+    def _get_new_crop_thumb(self, size):
+        # "Imports"
+        Image = self.field.db_field.rel.to
+        Thumb = Image._meta.get_field("thumbs").rel.to
+
+        box = Box(0, 0, self.width, self.height)
+        crop_box = Crop(box, self.path)
+
+        best_fit = size.fit_to_crop(crop_box, original_image=self.path)
+        fit_box = best_fit.box
+        crop_thumb = Thumb(**{
+            "name": size.name,
+            "width": fit_box.w,
+            "height": fit_box.h,
+            "crop_x": fit_box.x1,
+            "crop_y": fit_box.y1,
+            "crop_w": fit_box.w,
+            "crop_h": fit_box.h,
+        })
```

Comment:
I would be remiss if I didn't point out that the example you gave for how to use keyword arguments (thank you for that, by the way) produces the error [`E202:whitespace before ')'`](https://www.python.org/dev/peps/pep-0008/#pet-peeves). In order for it to comply, the closing parenthesis should either be on the same line as the last kwarg, or the last kwarg should have a trailing comma. Anecdotally, my sense is that the former is the more common style. In fact, I am potentially swayed by that style over `**{}` for argument passing, because then you don't have stray syntax interrupting the flow of code (`})`, or `)`)